packages/generator-nitro/generators/app/templates/package.json

```diff
@@ -47,7 +47,7 @@
     "build:webpack": "webpack --mode production --config config/webpack/webpack.config.prod.js",<% if (options.exporter) { %>
     "export": "npm run build && gulp export",<% } %>
     "\n# RUN ------- ": "",
-    "dashboard": "dash4",
+    "dashboard": "opener http://localhost:4444 && dash4",
```

Comment:
`&&` causes issues on some shells - maybe use npm-run-all

Reply:
What issues are you thinking about?
We have been using && in other scripts for years (see line before) without negative feedback.

Reply:
Hey @ernscht 

you are right sorry for the false alarm :)  
Not sure if this was fixed in the meantime but it works very well now for me even if my terminal does not support it:

<img width="277" alt="and_and" src="https://user-images.githubusercontent.com/4113649/68854145-c0c59480-06db-11ea-8ac3-ffdd4e05e3bf.png">